.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 npm-debug.log*
 /dist
+.DS_Store
```

Comment:
Can you put this in your global gitignore? This file should only be related to the project itself and not OS level files.